pkg/operator/controller.go

```diff
@@ -491,3 +499,33 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 
 	return nil
 }
+
+func (c *Controller) hasTrustedCA() bool {
+	_, err := os.Stat("/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem")
+	return err != nil
+}
+
+func (c *Controller) syncOperatorStatus(cr *imageregistryv1.Config, prevCR *imageregistryv1.Config) error {
+	cr.Status.ObservedGeneration = cr.Generation
+	statusChanged := !reflect.DeepEqual(prevCR.Status, cr.Status)
```

Comment:
Just a small improvement here, we could design this as:

```
if reflect.DeepEqual(prevCR.Status, cr.Status) {
        // No changes, just return.
        return nil
}
```
We have the whole function within an if statement otherwise.